governance/votes/Executive vote - August 22, 2024.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Template - [Executive Vote] LITE-PSM-USDC-A Phase 2 Setup - August 22, 2024
+summary: Update PSM State Variable in conduit contracts to MCD_LITE_PSM_USDC_A, USDC Migration from PSM-USDC-A to LITE-PSM-USDC-A, update PSM-USDC-A Fees, update PSM-USDC-A DC-IAM parameters, update MCD_LITE_PSM_USDC_A buf, update LITE-PSM-USDC-A DC-IAM parameters, GSM Delay update, update LitePSM Keeper Network Job.
+date: 2024-08-22T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] LITE-PSM-USDC-A Phase 2 Setup - August 22, 2024
+
+The Governance Facilitators, Dewiz, and Sidestream have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **actions** will occur within the Maker Protocol:
+
+- The PSM state variables in the conduit contracts will be updated to MCD_LITE_PSM_USDC_A.
+- Migrate **all but 200 million USDC reserves** from PSM-USDC-A to LITE-PSM-USDC-A.
+- PSM-USDC-A fees will be updated.
+- PSM-USDC-A DC-IAM parameters will be updated.
+- MCD_LITE_PSM_USDC_A `buf` will be increased.
+- MCD_LITE_PSM_USDC_A DC-IAM parameters will be updated.
```

Comment:
```suggestion
- LITE-PSM-USDC-A DC-IAM parameters will be updated.
```

Reply:
@blimpa, since you used PSM-USDC-A for the old PSM instead of MCD_PSM_USDC_A, I’ll apply this suggestion for consistency